graphstorm-processing/tests/test_dist_heterogenous_loader.py

```diff
@@ -116,6 +116,21 @@ def data_configs_with_label_fixture():
     return data_configs_dict
 
 
+@pytest.fixture(scope="function", name="data_configs_homogeneous")
+def data_configs_homogeneous_fixture():
+    """Create data configuration object that contain features and labels"""
+    config_path = os.path.join(
+        _ROOT, "resources/small_heterogeneous_graph/gsprocessing-homogeneous.json"
+    )
+
+    with open(config_path, "r", encoding="utf-8") as conf_file:
+        gsprocessing_config = json.load(conf_file)
+
+    data_configs_dict = create_config_objects(gsprocessing_config["graph"])
+
+    return data_configs_dict
```

Comment:
I think this dict is small enough to just define as an object, no need to read from a file? Let's only keep files for more complex configs.